portfolio/src/main/webapp/js/comments.js

```diff
@@ -15,7 +23,15 @@ function initCommentsSystem() {
  * and displays them in the comments section.
  */
 function loadComments() {
-  fetch('/comments').then(resp => {
+  const commentControl = document.getElementById("comment-control");
+  const loadUrl = 
+      new URL("/comments", `${location.protocol}//${location.hostname}`);
+  loadUrl.searchParams.set(
+      "numPerPage", commentControl.elements["numPerPage"].value);
+  loadUrl.searchParams.set(
+      "page", commentControl.elements["pageNum"].value)
+
+  fetch(loadUrl).then(resp => {
```

Comment:
nit: just noticed the spacing below here is funny, can we fix that either here or in another PR?

Should be 2 spaces after { just like all the other functions/if statements etc.

Reply:
Oh okay. Then that is probably the case for all of the fetch requests in this file. Will update.